src/components/AuthContainer.js

```diff
@@ -38,6 +39,50 @@ export default class AuthContainer extends Component {
       }, () => {
         console.warn('Error looking up user status')
       })
+
+      Rawls.listBillingProjects().then(
+        billingProjects => {
+          this.billingProjects = billingProjects
+          this.checkClusters()
+        },
+        failure => Utils.log('Error fetching billing projects', failure)
```

Comment:
TODO for me: Write a story to capture exposing errors to the user and logging them to New Relic.